### PR TITLE
Minor updates to tutorial part 0 & 1

### PR DESCRIPTION
**Pull request by mdole**
Hey folks, was just running through the tutorial for the first time and noticed a few things that I thought should maybe be updated. Quick breakdown:
- Update link to resource on learning how the web works (existing link directs to a site with no information that tries to sell a coding course)
- Update node version requirement—when I was installing with node v6, I got this error: 
`error serve@6.5.8: The engine "node" is incompatible with this module. Expected version ">=7.6.0".`
- Add `npm install` before first call to `gatsby develop`

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 5c0e0c7905844bedc2dcd9471dd18b9d3e161eed

https://app.netlify.com/sites/using-postcss-sass/deploys/5b5357f702ed832bb71004d4

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -62,9 +62,10 @@ nothing is happening at first; be patient!
 Now let's try running Gatsby!
 
 Gatsby has a built-in development server. Let's start it up by running the
-following command:
+following commands:
 
 ```sh
+npm install
```
Comment (Raincal) [inline docs/tutorial/part-one/index.md]: Gatsby will automatically install the needed npm packages when using the `gatsby new` command.

File: docs/tutorial/part-one/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ You should see something like:
 
 ![Check if node.js/npm is installed](check-versions.png)
 
-Gatsby supports versions of Node back to v6 and npm to v3.
+Gatsby supports versions of Node back to v7.6 and npm to v3.
```
Comment (Raincal) [inline docs/tutorial/part-one/index.md]: I just tried v7.x
When I open localhost:9000 I get the following error:
```
TypeError: promisify is not a function

  - index.js:21 getHandlers
    [tutorial-part-one]/[serve-handler]/src/index.js:21:8

  - index.js:499 module.exports
    [tutorial-part-one]/[serve-handler]/src/index.js:499:19

  - serve.js:16 Server.http.createServer
    [tutorial-part-one]/[gatsby]/dist/commands/serve.js:16:57
```
`promisify` is a new utility function from Node 8.
So the minimum Node.js version required for Gatsby should be v8.x now.

Comment (m-allanson) [reply docs/tutorial/part-one/index.md]: Gatsby should support all [LTS releases of Node](https://github.com/nodejs/Release). If there are errors with Node 6 or 7 then that's an issue with Gatsby that should be fixed.

**Comment by m-allanson** [comment]
Thanks @mdole and @Raincal!

@mdole I'm curious if you were following the tutorial on:

- gatsbyjs.org - which covers Gatsby v1 and is served from branch `v1`?

or 

- next.gatsbyjs.org - which covers the v2 beta and is served from branch `master`?

It looks like the 'how the web works' link was updated in #6614, but only for next.gatsbyjs.org. Would you like to open a new PR that makes the same change for `v1`?

I think @Raincal is right in that your other changes don't apply here, so I'm going to close this PR. However, it looks like your PR surfaced a bug - Gatsby should be supporting Node.js v6. Thanks for revealing that! I'll open an issue to get that resolved.

**Comment by m-allanson** [comment]
Here's the Node v6 issue: https://github.com/gatsbyjs/gatsby/issues/6671

**Comment by mdole** [comment]
@m-allanson and @Raincal thanks for responding and reviewing! I was doing the tutorial on gatsbyjs.org, to answer your question. And sure, I'll make a fresh PR to update the 'how the web works' link on v1

**Pull request closed without merging**